data/engines.txt

```diff
@@ -492,3 +492,16 @@ outfit "AR120 Reverse Thruster"
 	"reverse thrusting energy" 1.8
 	"reverse thrusting heat" 3.5
 	description "For pilots who want to be able to pull fancy tricks that most ships are not capable of emulating, a reverse thruster allows a ship to accelerate backwards, without needing to turn around. Because a reverse thruster must be facing forwards, it must be installed in the weapon section of your ship instead of the engine section."
+
+outfit "Stationing Engines"
+	category "Engines"
+	"cost" 75000
+	thumbnail "outfit/ionic afterburner"
+	"mass" 8
+	"outfit space" -10
+	"engine capacity" -8
```

Comment:
i had intended for the outfit to use more outfit space than engine capacity - missed the tonnage point during revision - thanks!